src/components/CheckedLabel/types.ts

```diff
@@ -0,0 +1,14 @@
+import {PropsWithChildren, ReactNode} from 'react';
+import {DOMProps, QAProps} from '../types';
+
+export type Sizes = 'm' | 'l';
+
+export type Props = PropsWithChildren<{
+    labelClassName?: string;
```

Comment:
simply `className`, there is no more elements to add class to

Reply:
There is also `className`, inherited from `DOMProps`